tests/shell/shell_recorder/globalplugins.esr

```diff
@@ -1,32 +0,0 @@
-Mountpoint: system/
-DiffType: File
-
-
-< set system/elektra/globalplugins ""
```

Comment:
stderr is okay, see `src/plugins/timeofday/timeofday.c`

The output is a bit strange because the get commands are missing in the output? So seems like a bug in the global hooks? There have been quite some changes there.

So you can leave it in this PR as suggested. But can you please create an issue that the test case is not working anymore?

Btw. timeofday does not even have the `infos/status` to be mounted globally. Although it should have. I fixed it in bbc2980656f2bd561ca1dc1d79e9fbf033c50f7a